homeassistant/components/ozw/light.py

```diff
@@ -137,6 +137,24 @@ def color_temp(self):
         """Return the color temperature."""
         return self._ct
 
+    @property
+    def min_mireds(self):
+        """Return the coldest color_temp that this light supports."""
+        if self.values.max_kelvin:
+            self._min_mireds = color_util.color_temperature_kelvin_to_mired(
```

Comment:
Should we move the update to `_calculate_rgb_values`? Otherwise that method won't have the updated values yet during a state update.

Reply:
Yes the calculation should be done in `_calculate_rgb_values`, I forgot to mention this. Sorry @tradiuz